playbooks/image.yaml

```diff
@@ -0,0 +1,80 @@
+# This playbook ensures that a base image is up to date with all of the required settings
+
+- name: Launch an instance to build images
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  tasks:
+    - name: Create a persistent disk for the image
+      gce_pd:
+        service_account_email: "{{ gce_service_account }}"
+        credentials_file: "{{ playbook_dir + '/files/' + gcs_registry_keyfile }}"
+        project_id: "{{ gce_project_id }}"
+        zone: "{{ gce_zone_name }}"
+        name: "{{ provision_prefix }}build-image-instance"
+        disk_type: pd-ssd
+        image: "{{ provision_gce_base_image | default('https://www.googleapis.com/compute/v1/projects/rhel-cloud/global/images/rhel-7-v20170110') }}"
+        size_gb: 10
```

Comment:
This is probably the parameter we end up tweaking the most for image creation, next to RPM dependency list.